gems/activerecord/6.0.3.2/_test/Steepfile

```diff
@@ -0,0 +1,25 @@
+target :test do
+  signature "."
+  check "."
+
+  repo_path "../../../"
+
+  library "pathname"
```

Comment:
Required libraries. This is the same list as the one found on the [pock/rbs_rails](https://github.com/pocke/rbs_rails#steep-integration) README.